README.md

```diff
@@ -82,7 +111,7 @@ pip3 install .
 The camp tool takes a YANG representation of an ADM for a network protocol as
 input and calls each of the included generators to generate files for the ADM.
 
-<!-- > The included `template.json` provides an example of how a JSON ADM should be
+<!-- > The included `template.yang` provides an example of how a YANG ADM should be
```

Comment:
Use the term "ADM module" instead of "YANG ADM" to mean "an ADM encoded as a YANG module file"